modules/objc/generator/gen_objc.py

```diff
@@ -1544,6 +1542,39 @@ def sanitize_documentation_string(doc, type):
             break
     return "/**\n " + "\n ".join(lines) + "\n */" if hasValues else ""
 
+# copytree from shutil cannot handle an error when directory exists in Python < 3.8
+def _mkdir(newdir):
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir

with `parents=True`

Reply:
Added it to the function.

Reply:
I mean that we could replace this function implementation with `pathlib.Path.mkdir` call (available since Python 3.4).